modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -550,6 +660,173 @@ private void _assertGetObjectDefinitionsPageWithFilter(
 			expectedObjectDefinitions, (List<ObjectDefinition>)page.getItems());
 	}
 
+	private void _assertObjectValidationRule(
+		String expectedObjectFieldExternalReferenceCode,
+		ObjectValidationRule expectedObjectValidationRule,
+		ObjectValidationRule actualObjectValidationRule) {
+
+		Assert.assertEquals(
+			expectedObjectValidationRule.getActive(),
+			actualObjectValidationRule.getActive());
+		Assert.assertEquals(
+			expectedObjectValidationRule.getEngine(),
+			actualObjectValidationRule.getEngine());
+		Assert.assertEquals(
+			expectedObjectValidationRule.getErrorLabel(),
+			actualObjectValidationRule.getErrorLabel());
+		Assert.assertEquals(
+			expectedObjectValidationRule.getExternalReferenceCode(),
+			expectedObjectValidationRule.getExternalReferenceCode());
+		Assert.assertEquals(
+			expectedObjectValidationRule.getName(),
+			actualObjectValidationRule.getName());
+		Assert.assertEquals(
+			expectedObjectValidationRule.getOutputType(),
+			actualObjectValidationRule.getOutputType());
+		Assert.assertEquals(
+			expectedObjectValidationRule.getScript(),
+			actualObjectValidationRule.getScript());
+
+		if (StringUtil.equals(
+				actualObjectValidationRule.getOutputType(
+				).getValue(),
+				ObjectValidationRuleConstants.OUTPUT_TYPE_FULL_VALIDATION)) {
+
+			Assert.assertTrue(
+				ArrayUtil.isEmpty(
+					actualObjectValidationRule.
+						getObjectValidationRuleSettings()));
+		}
+		else if (StringUtil.equals(
+					actualObjectValidationRule.getOutputType(
+					).getValue(),
```

Comment:
@pedro-oliveira446 please [avoid chaining](https://liferay.atlassian.net/wiki/spaces/ENGR/pages/1511752519/Java+conventions+in+Liferay#%5BinlineExtension%5DAvoid-chaining).. You can use the `getOutputTypeAsString()` instead